src/Func.h

```diff
@@ -1532,6 +1533,9 @@ class Func {
      * different values at different times or on different machines. */
     Func &allow_race_conditions();
 
+    /** Issue atomic store for this Func. This allows parallelization on arbitrary RVars. */
```

Comment:
We'll need to substantially augment this docstring before landing.

Reply:
Augmented. Please take a look.